content/docs/introduction/faq.md

```diff
@@ -74,12 +86,11 @@ For cases where you must push, we offer the [Pushgateway](/docs/instrumenting/pu
 Short answer: Don't! Use something like [Grafana Loki](https://grafana.com/oss/loki/) or [OpenSearch](https://opensearch.org/) instead.
 
 Longer answer: Prometheus is a system to collect and process metrics, not an
-event logging system. The Raintank blog post
-[Logs and Metrics and Graphs, Oh My!](https://blog.raintank.io/logs-and-metrics-and-graphs-oh-my/)
+event logging system. The Grafana blog post
+[Logs and Metrics and Graphs, Oh My!](https://grafana.com/blog/2016/01/05/logs-and-metrics-and-graphs-oh-my/)
 provides more details about the differences between logs and metrics.
 
-If you want to extract Prometheus metrics from application logs, Google's
-[mtail](https://github.com/google/mtail) might be helpful.
+If you want to extract Prometheus metrics from application logs, Grafana Loki is designed for just that. See Loki's [metric queries](https://grafana.com/docs/loki/latest/logql/metric_queries/) documentation.
```

Comment:
I prefer to recommend mtail. Mtail is lower footprint, as if I "want to extract Prometheus metrics from application logs", that would be my go to solution.